src/createAnimatedComponent.tsx

```diff
@@ -409,11 +472,64 @@ export default function createAnimatedComponent(
       }
     }
 
+    _attachInlineStyles() {
+      const newInlineStyles: StyleProps = getInlineStylesFromProps(this.props);
+      const hasChanged = inlineStylesHasChanged(
+        newInlineStyles,
+        this._inlineStyles
+      );
+
+      if (hasChanged) {
+        const sharableViewDescriptors =
+          this._inlineStylesViewDescriptors.sharableViewDescriptors;
+
+        const maybeViewRef = NativeReanimatedModule.native
+          ? undefined
+          : ({ items: new Set([this]) } as ViewRefSet<any>); // see makeViewsRefSet
```

Comment:
we can follow similar approach with view descriptors and only define them here, no?